cartoframes/data/dataset.py

```diff
@@ -185,13 +185,14 @@ def dataset_info(self):
 
         return self._strategy.dataset_info
 
-    def update_dataset_info(self, privacy=None, name=None):
+    def update_dataset_info(self, privacy=None, table_name=None):
         """Update/change Dataset privacy and name
 
         Args:
           privacy (str, optional): One of DatasetInfo.PRIVATE,
```

Comment:
For docs purposes, these could be crosslinked with this syntax (I think this is correct):
:py:attr:`DatasetInfo.PRIVATE`